libreyolo/models/rfdetr/nn.py

```
@@ -854,6 +877,30 @@ def __init__(
+        )
+        kp_kwargs = {}
+        if pose:
+            kp_kwargs = dict(
+                use_grouppose_keypoints=True,
+                num_keypoints_per_class=self.num_keypoints_per_class,
+                grouppose_keypoint_dim_downscale=1,
+                keypoint_cross_attn=True,
+                inter_instance_kp_attn=False,
+                dual_projector=True,
+                dual_projector_kp_only=True,
+            )
```

Ruff (0.15.17)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)